lxc/storage.go

```diff
@@ -88,6 +88,10 @@ lxc storage delete [<remote>:]<pool>
 lxc storage edit [<remote>:]<pool>
     Edit storage pool, either by launching external editor or reading STDIN.
 
+lxc storage <pool-name> btrfs.mount_options "<mount option>,<mount option>,<mount option>"
+    add btrfs mount options to btrfs storage pool
+    EXAMPLE: lxc storage <pool-name> btrfs.mount_options "rw,nospace_cache,lazytime,user_subvol_rm_allowed,autodefrag"
+
```

Comment:
> you'd need to get a working Go dev environment for that to work
it may not be the last time I run into this. So maybe it's worth the effort? If there just was a container template/image with exactly that ...

seems it goes beyond installing `golang` . What else do I need?